.github/settings.yaml

```diff
@@ -0,0 +1,156 @@
+  - { name: '🔧 maintenance',            color: '#c2f759', description: '' }
+  - { name: '🚀 enhancement',            color: '#84b6eb', description: '' }
+  - { name: '🚫 wontfix',                color: '#ffffff', description: '' }
+  - { name: '🛟 help wanted',            color: '#128A0C', description: '' }
+
+# Collaborators: give specific users access to this repository.
+# See https://docs.github.com/en/rest/reference/repos#add-a-repository-collaborator for available options
+collaborators:
+  - username: totallyzen
+    permission: maintain
+  - username: alexanderankin
+    permission: maintain
+  - username: kiview
+    permission: admin
+  #- username: testcontainersbot
+  #  permission: write
+
+  # Note: `permission` is only valid on organization-owned repositories.
+  # The permission to grant the collaborator. Can be one of:
+  # * `pull` - can pull, but not push to or administer this repository.
+  # * `push` - can pull and push, but not administer this repository.
+  # * `admin` - can pull, push and administer this repository.
+  # * `maintain` - Recommended for project managers who need to manage the repository without access to sensitive or destructive actions.
+  # * `triage` - Recommended for contributors who need to proactively manage issues and pull requests without write access.
+
+# See https://docs.github.com/en/rest/reference/teams#add-or-update-team-repository-permissions for available options
+teams:
+  # Please make sure the team already exist in the organization, as the repository-settings application is not creating them.
+  # See https://github.com/repository-settings/app/discussions/639 for more information about teams and settings
+  # - name: go-team
+  #   # The permission to grant the team. Can be one of:
+  #   # * `pull` - can pull, but not push to or administer this repository.
+  #   # * `push` - can pull and push, but not administer this repository.
+  #   # * `admin` - can pull, push and administer this repository.
+  #   # * `maintain` - Recommended for project managers who need to manage the repository without access to sensitive or destructive actions.
+  #   # * `triage` - Recommended for contributors who need to proactively manage issues and pull requests without write access.
+  #   permission: admin
+  - name: oss-maintainers
+    permission: admin
+
+branches:
+  - name: main
+    # https://docs.github.com/en/rest/reference/repos#update-branch-protection
+    # Branch Protection settings. Set to null to disable
+    protection:
+      # Required. Require at least one approving review on a pull request, before merging. Set to null to disable.
+      required_pull_request_reviews:
+        # The number of approvals required. (1-6)
+        required_approving_review_count: 1
+        # Dismiss approved reviews automatically when a new commit is pushed.
+        dismiss_stale_reviews: false
+        # Blocks merge until code owners have reviewed.
+        require_code_owner_reviews: false
```

Comment:
I think we do need to make this `true` and add a `.github/CODEOWNERS` with this at the least:

```
.github/CODEOWNERS @testcontainers/oss-matinainers
.github/settings.yaml @testcontainers/oss-matinainers
```

Why we need this?
- the app that is running this sync process has an explicit security warning about this
- their suggested workaround is the above mentioned requirement plus the `CODEOWNERS` change